src/test/java/com/minecolonies/coremod/colony/ColonyTest.java

```diff
@@ -91,33 +80,36 @@ public void setup() throws NoSuchFieldException, IllegalAccessException
     public void testToggleAnything()
     {
         colony.setName("blahsColony");
+        colony.writeToNBT(new NBTTagCompound());
         assertNotEquals(null, colony.getColonyTag());
     }
 
     @Test
     public void testToggleOverManager()
     {
         colony.getBarbManager().setCanHaveBarbEvents(true);
+        colony.writeToNBT(new NBTTagCompound());
         assertNotEquals(null, colony.getColonyTag());
     }
 
     @Test
     public void testToggleInLoad()
     {
-        colony.markDirty();
-        NBTTagCompound compound = colony.getColonyTag();
-        Colony test = Colony.loadColony(compound, world);
+        colony.writeToNBT(new NBTTagCompound());
+        final NBTTagCompound compound = colony.getColonyTag();
+        final Colony test = Colony.loadColony(compound, world);
         assertNotEquals(null, test.getColonyTag());
```

Comment:
Should there also be some sort of check that compares `test` to `colony`? or `compound` to `test.getColonyTag`?